package.json

```diff
@@ -5,9 +5,8 @@
     "access": "public"
   },
   "dependencies": {
-    "@datapunt/asc-assets": "0.18.2-alpha.145",
-    "@datapunt/asc-core": "0.18.1",
-    "@datapunt/asc-ui": "0.18.2-alpha.4",
+    "@datapunt/asc-assets": "0.18.3-alpha.8",
```

Comment:
can we move these to peerDependences?

Reply:
I've updated, @jonkoops @aditudorache do you agree with this?
https://github.com/Amsterdam/amsterdam-react-maps/compare/995cafb42fdcd85eb336bbebde29e95119dd94bb..a52a3f3a69d46e14ae294900b97d26c8608f1d6e